.probo.yaml

```diff
@@ -69,7 +69,7 @@ steps:
       - chown www-data:www-data -R /var/www/html/sites/default/files
       - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -y -n pm:install environment_indicator
       - chmod 777 /var/www/html/sites/default/settings.php
-      - echo -e "\$config['environment_indicator.indicator']['name'] = '$BRANCH_NAME (ProboCI)';\n\$config['environment_indicator.indicator']['bg_color'] = '#440027';\n\$config['environment_indicator.indicator']['fg_color'] = '#ffffff';" >> /var/www/html/sites/default/settings.php
+      - echo -e "\$config['environment_indicator.indicator']['name'] = '$BRANCH_NAME (ProboCI)';\n\$config['environment_indicator.indicator']['bg_color'] = '#440027';" >> /var/www/html/sites/default/settings.php
```

Comment:
Is white the default color or something?

Reply:
This core change was causing problems with that setting:
https://www.drupal.org/project/drupal/issues/3097907

Reply:
Reopening this conversation since I am noticing a potentially related issue in Pantheon with the text color for active items in the admin toolbar. This is not happening in Probo, but only in Pantheon multidevs created for testing this PR:

<img width="343" alt="image" src="https://github.com/az-digital/az_quickstart/assets/74572157/00ccec26-3368-4771-9a2d-e5f2246768d8">
<p></p>

For some reason, this style attribute is being included with the `<a>` tag for each `toolbar-tab` element:
<p></p>

```
style="color: rgb(255, 255, 255);"
```

Reply:
> Reopening this conversation since I am noticing a potentially related issue in Pantheon with the text color for active items in the admin toolbar. This is not happening in Probo, but only in Pantheon multidevs created for testing this PR:
> 
> For some reason, this style attribute is being included with the `<a>` tag for each `toolbar-tab` element:
> 
> ```
> style="color: rgb(255, 255, 255);"
> ```

@bberndt-uaz There is a change to our Pantheon `settings.upstream.php` file in this PR that should address that issue there:
https://github.com/az-digital/az-quickstart-pantheon/pull/251

Reply:
Got it! I'll resolve this conversation again.

Reply:
Thanks!